Source/WebCore/dom/Element.cpp

```diff
@@ -733,7 +733,7 @@ AtomString Element::getAttributeForBindings(const QualifiedName& name, ResolveUR
         return attribute->value();
     }
 
-    ASSERT_NOT_REACHED();
+    RELEASE_ASSERT_NOT_REACHED();
```

Comment:
Can we write this differently so it silences the warning but doesn't add release time code to this hot function?

Reply:
We could use IGNORE_RETURN_TYPE_WARNINGS_BEGIN/END